### PR TITLE
Add basic structure to the page and relevant informations with HTML 

### PR DESCRIPTION
**Pull request by jiminbyun**
From previously empty page with just a name, in this change, more contents to the pages are added including description about myself, experience, and external links. It also has new second page about the pet with images. Each section divided based on content was seperated with div, so that later each section can be easily styled differently.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -7,12 +7,65 @@
...
+        <button onclick="addRandomGreeting()">...</button>
+        <div id="greeting-container"></div>
+    </div>
+    <div id="exp">
+        <h2>Experiences</h2>
+        <div id="ta">
+            <h3>Teaching Assistant, Spring 2020-Present </h3>
+            <p>Carnegie Mellon, Concepts of A.I, A.I Representation and Problem Solving</p>
+            <ul>
+                <li>Teach various artificial intelligence methods and algorithms to implement them.</li>
```
Comment (darrnshn) [inline portfolio/src/main/webapp/index.html:43]: nit: you may omit the closing </li> tag. See https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -7,12 +7,65 @@
...
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="home">
+        <h1>Jimin Byun</h1>
```
Comment (darrnshn) [inline portfolio/src/main/webapp/index.html:11]: nit: please use 2 spaces for indentation: https://google.github.io/styleguide/htmlcssguide.html#Indentation

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -7,12 +7,65 @@
...
+    <div id="random">
+        <h2>Random-Random Information</h2>
+        <p>Click here to get a random</p>
+        <button onclick="addRandomGreeting()">...</button>
+        <div id="greeting-container"></div>
+    </div>
+    <div id="exp">
+        <h2>Experiences</h2>
+        <div id="ta">
+            <h3>Teaching Assistant, Spring 2020-Present </h3>
```
Comment (darrnshn) [inline portfolio/src/main/webapp/index.html:40]: nit: remove trailing whitespace after "Present"

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -7,12 +7,65 @@
...
+            <p>Carnegie Mellon, Concepts of A.I, A.I Representation and Problem Solving</p>
+            <ul>
+                <li>Teach various artificial intelligence methods and algorithms to implement them.</li>
+                <li>Hold weekly recitations and office hours to help with programming and conceptual questions.</li>
+                <li>Create questions for quiz and exams and pretest programming homeworks</li>
+            </ul>
+        </div>
+        <div id="patent_ra">
+            <h3>Patent Reading Group, Research Assistant, Spring 2020 </h3>
```
Comment (darrnshn) [inline portfolio/src/main/webapp/index.html:49]: nit: remove trailing space after 2020

File: portfolio/src/main/webapp/woof.html [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="UTF-8">
```
Comment (darrnshn) [inline portfolio/src/main/webapp/woof.html:5]: nit: 2 space indent for this file as well

File: portfolio/src/main/webapp/woof.html [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+	<title>My Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
```
Comment (darrnshn) [inline portfolio/src/main/webapp/woof.html:10]: nit: no need for this blank line

File: portfolio/src/main/webapp/woof.html [hunk 7]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+
```
Comment (darrnshn) [inline portfolio/src/main/webapp/woof.html:3]: nit: no need for this blank line

File: portfolio/src/main/webapp/woof.html [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+<head>
+	<meta charset="UTF-8">
+	<title>My Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+    <h1> Woof. </h1>
+	<img src="/images/pet/IMG-0057.JPG" alt="Beef jerky loving dog" height="500">
```
Comment (darrnshn) [inline portfolio/src/main/webapp/woof.html:13]: super nit: usually we keep presentation logic like "height" in CSS: https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns
but you haven't got up to that part yet, so maybe you can put a HTML comment above this line to say "TODO: Move height to CSS" or something.

File: portfolio/src/main/webapp/woof.html [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+		Jindo dogs are typically hunter dogs so are very smart and loyal, but our dog is different.
+		He doesn't know whether there is a stranger in the house until someone reaches to our front door.
+		He often skips meal because he is scared of mouse near his food tray (He eats right after we remove mouse for
+		him).
+		He also can't play catch because when I throw a ball, he runs to it and fetches it, but instead of giving back
+		he constantly runs around with it in his mouth until he gets tired, which then he just lies down.
+		Still, he was the best addition to our family. 
+    </p>
+</body>
+
```
Comment (darrnshn) [inline portfolio/src/main/webapp/woof.html:26]: nit: remove blank line

File: portfolio/src/main/webapp/woof.html [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+	<meta charset="UTF-8">
+	<title>My Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+    <h1> Woof. </h1>
+	<img src="/images/pet/IMG-0057.JPG" alt="Beef jerky loving dog" height="500">
+	<p> Meet Dukgu, traditional Korean Jindo dog. He just turned 2 years so he is very young.
```
Comment (darrnshn) [inline portfolio/src/main/webapp/woof.html:14]: Dukgu is very cute

File: portfolio/src/main/webapp/woof.html (hunk 8 above)
Comment (darrnshn) [inline portfolio/src/main/webapp/woof.html:13]: I noticed that you only used IMG-0057, but you've uploaded several other images as well. It's best to keep pull requests small, so let's only upload the photos that you need in this PR.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (jiminbyun) [reply portfolio/src/main/webapp/index.html]: I tried deleting all the optional tags (including closing tags for body and head). Not sure whether I should delete start tags for heads and body too.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (jiminbyun) [reply portfolio/src/main/webapp/index.html]: Done!

File: portfolio/src/main/webapp/woof.html (hunk 8 above)
Comment (jiminbyun) [reply portfolio/src/main/webapp/woof.html]: Oh okay sounds good

File: portfolio/src/main/webapp/woof.html (hunk 10 above)
Comment (jiminbyun) [reply portfolio/src/main/webapp/woof.html]: haha thanks

File: portfolio/src/main/webapp/woof.html [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+  <h1> Woof. </h1>
+  <!-- TODO: Move height to CSS -->
+  <img src="/images/pet/IMG-0057.JPG" alt="Beef jerky loving dog" height="500">
+  <p> Meet Dukgu, traditional Korean Jindo dog. He just turned 2 years so he is very young.
+      He was very small when he first came, and our family sort of thought he would stay that way for a while, but he grew exponentially every week; still he is super adorable.
+      Jindo dogs are typically hunter dogs so are very smart and loyal, but our dog is different.
+      He doesn't know whether there is a stranger in the house until someone reaches to our front door.
+      He often skips meal because he is scared of mouse near his food tray (He eats right after we remove mouse for him).
+      He also can't play catch because when I throw a ball, he runs to it and fetches it, but instead of giving back he constantly runs around with it in his mouth until he gets tired, which then he just lies down.
+      Still, he was the best addition to our family.
```
Comment (darrnshn) [inline portfolio/src/main/webapp/woof.html:17]: nit: the diff shows that you don't have a newline at the end. I think usually we always have a trailing newline character: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

**Pull request merged**